tests/Makefile

```diff
@@ -16,15 +16,15 @@ GOLANG_VERSION ?= 1.12
 GOPATH ?= $(HOME)/go
 
 # Comma seperated items within {} for more than one file
-EXCLUDE ?= gcp-credentials-admission-webhook_test.go
+EXCLUDE ?= {jupyter_test.go,profiles_test.go}
 
 export GO111MODULE = on
 export GO = go
 
 all: test
 
 generate: clean
-	@cd .. && hack/gen-test-targets.sh && rm tests/$(EXCLUDE)
+	@cd .. && hack/gen-test-targets.sh && [ -n "$(EXCLUDE)" ] && rm -f tests/$(EXCLUDE) && rm -f tests/jupyter_test.go && rm -f tests/profiles_test.go || cd -
```

Comment:
this can now be what's on master
`        @cd .. && hack/gen-test-targets.sh && [ -n "$(EXCLUDE)" ] && rm -f tests/$(EXCLUDE)`